src/components/RetryApiRegistration/RetryApiRegistration.js

```diff
@@ -60,7 +65,7 @@ class RetryApiRegistration extends React.Component<Props> {
     return (
       <Container center>
         {walletState !== REGISTRATION_FAILED && (
-          <Loader messages={['Registering on backend']} />
+          <Loader messages={[t('auth.loadingMessage.registering')]} />
```

Comment:
Why one call starts with "auth:" and this "auth."? Maybe all can be just "auth." if both works?